packages/platform-app/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "@frontify/platform-app",
+    "version": "0.0.2",
+    "description": "Provides the types for the Apps",
+    "sideEffects": false,
+    "main": "dist/index.umd.js",
+    "unpkg": "dist/index.umd.js",
+    "jsdelivr": "dist/index.umd.js",
+    "module": "dist/index.es.js",
+    "types": "dist/index.d.ts",
+    "author": "Frontify Developers <developers@frontify.com>",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Frontify/brand-sdk",
+        "directory": "packages/platform-app"
+    },
+    "publishConfig": {
+        "access": "public"
+    },
+    "scripts": {
+        "build": "vite build",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "prettier": "prettier --check .",
+        "prettier:fix": "prettier --write .",
+        "typecheck": "tsc --noEmit",
+        "test": "vitest run",
+        "test:watch": "vitest",
+    },
+    "devDependencies": {
+        "@frontify/app-bridge": "^3.1.0",
+        "@frontify/eslint-config-react": "0.16.2",
+        "@types/react": "^18.2.39",
+        "eslint": "^8.54.0",
+        "eslint-plugin-notice": "^0.9.10",
+        "prettier": "^3.1.0",
+        "typescript": "^5.3.2",
+        "vite": "^5.0.4",
+        "vite-plugin-dts": "^3.6.3",
+        "vitest": "^0.34.6",
+        "react": "^18",
+        "react-dom": "^18"
+
+    },
+    "dependencies": {
+        "@frontify/sidebar-settings": "^0.8.0"
```

Comment:
```suggestion
        "@frontify/sidebar-settings": "workspace:^"
```

Reply:
I tried a publish of the package to see if everything works and it complained with the workspace setting

Reply:
With `pnpm publish`?

Reply:
used `npm publish`, but i also had an error in the package.json, ill change it back and try again 🦖